modules/gapi/src/api/kernels_video.cpp

```diff
@@ -0,0 +1,47 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2020 Intel Corporation
+
+
+#include "precomp.hpp"
+
+#include <opencv2/gapi/video.hpp>
+
+namespace cv { namespace gapi {
+namespace
+{
+    using namespace video;
+} // namespace
```

Comment:
Why is anonymous namespace required here?

Reply:
Ih, it is a mistake, the file is .cpp so there is no need in this.. fixed